### PR TITLE
Add readme in english

### PR DESCRIPTION
**Pull request by RomanKonovalov**
Add readme in english

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
-# eighty
+# Eighty
+
+## About Eighty
+
+Backgrounds of the project were to learn to use new technologies, to participate in development of real applications, to gain experience of work in team.
+
+Eighty is a web-application with one page, that works through REST-service with graph-oriented database. This is a base of knowledge with questions and answers on different topics that lets to use the application as self-knowledge, testing and as questions for interview.
```
Comment (RomanKonovalov) [inline README.md]: Maybe: Eighty is a single page application...

Comment (RomanKonovalov) [inline README.md]: so graph database

**Comment by UladzimirKapylou** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+Goals at the beginning of the project:
+
+  *  All the questions are divided by topics that are saved in the tree.
+  *  User can add new questions, topics, update and delete existing questions and topics.
+  *  There is a filter by tags, contents.
+  *  Every user can increase question rating.
+
+## Used technologies
+  
+A lot of technologies from the widely used to highly specialized are used in project. Graph-oriented database Neo4j, Spring Framework (REST-service and Spring Data) are used as back-end. Bootstrap, jQuery, AngularJS are used as front-end. Application testing is based on JUnit, Karma, Protractor. Project building performs using Maven, NodeJS, Bower. Swagger is used for API project documentation.
```
Comment (RomanKonovalov) [inline README.md]: Project building performs using Maven and Grunt, package management - using Npm and Bower.

**Comment by RomanKonovalov** [comment]
:+1:

**Pull request merged**